packages/contract-wrappers/src/contract_wrappers/token_wrapper.ts

```diff
@@ -414,6 +414,7 @@ export class TokenWrapper extends ContractWrapper {
         );
         return logs;
     }
+    // tslint:disable:no-unused-variable
```

Comment:
@fragosti this will disable this rule for the rest of the file... I don't think that's what we want right?

Reply:
🤦‍♂️